js/components/facets/temporal-coverage.vue

```diff
@@ -34,6 +34,7 @@
 <script>
 import Calendar from 'components/calendar.vue';
 import moment from 'moment';
+import log from 'logger';
```

Comment:
Oups, forgot to remove (I started with logging first and this how I remember why the catch blocks where empty)